lib/modern_searchlogic/railtie.rb

```diff
@@ -2,7 +2,7 @@ module ModernSearchlogic
   class Railtie < Rails::Railtie
     initializer 'modern_searchlogic.setup_activerecord' do
       ActiveSupport.on_load(:active_record) do
-        ActiveRecord::Base.__send__(:include, ModernSearchlogic::ActiveRecordMethods)
+        ModernSearchlogic::ActiveRecordMethods.install
```

Comment:
Now that we have to consider both ActiveRecord::Base and ActiveRecord::Relation functionality move the calls inside of an encapsulated method on the ActiveRecordMethods module so we can keep all of the what-to-include logic together in a single spot.